modules/tooling-support/src/main/java/org/mule/runtime/module/tooling/internal/artifact/value/ValueProviderExecutor.java

```diff
@@ -87,9 +89,9 @@ public ValueResult resolveValues(ParameterizedModel parameterizedModel,
       if (cause instanceof ValueResolvingException) {
         failureBuilder.withFailureCode(((ValueResolvingException) cause).getFailureCode());
       }
```

Comment:
same here

Reply:
No, actually the way the connector has to return a "failure" is by throwing this exception which has a an error code. If any other exception is thrown we should propagate it but this transformation and exception handling is needed here.